app/src/lib/create-branch.ts

```diff
@@ -0,0 +1,39 @@
+import { TipState, Tip } from '../models/tip'
+import { StartPoint, Branch } from '../models/branch'
+
+type BranchInfo = {
```

Comment:
Any reason why a type alias was chosen over an interface?

Reply:
- convenience (it's not exported and was relatively trivial)
- my growing dislike of the `I` prefix on interfaces that this will require, and my growing preference of just using `type` unless we're looking to do object-oriented work in the codebase